src/app/(sidebar)/transaction/fee-bump/page.tsx

```diff
@@ -1,5 +1,200 @@
 "use client";
 
+import { useState } from "react";
+import { Button, Card, Icon, Text } from "@stellar/design-system";
+
+import { useStore } from "@/store/useStore";
+
+import { validate } from "@/validate";
+
+import { txHelper, FeeBumpedTxResponse } from "@/helpers/txHelper";
+
+import { useRouter } from "next/navigation";
+
+import { Box } from "@/components/layout/Box";
+import { PositiveIntPicker } from "@/components/FormElements/PositiveIntPicker";
+import { PubKeyPicker } from "@/components/FormElements/PubKeyPicker";
+import { SdsLink } from "@/components/SdsLink";
+import { TxResponse } from "@/components/TxResponse";
+import { ValidationResponseCard } from "@/components/ValidationResponseCard";
+import { XdrPicker } from "@/components/FormElements/XdrPicker";
+
 export default function FeeBumpTransaction() {
-  return <div>Fee Bump Transaction</div>;
+  const router = useRouter();
+  const { network, transaction } = useStore();
+  const {
+    feeBump,
+    updateBaseFeeSource,
+    updateBaseFeeBase,
+    updateBaseFeeInnerXdr,
+    updateSignActiveView,
+    updateSignImportXdr,
+    resetBaseFee,
+  } = transaction;
+  const { source_account, fee, innerTxXdr } = feeBump;
+
+  // Sign tx status related
+  const [feeBumpedTx, setFeeBumpedTx] = useState<FeeBumpedTxResponse>({
+    errors: [],
+    xdr: "",
+  });
+  const [xdrError, setXdrError] = useState<string>("");
+  const [sourceErrorMessage, setSourceErrorMessage] = useState<string>("");
+  const [baseErrorMessage, setBaseErrorMessage] = useState<string>("");
+
+  const onXdrChange = (value: string) => {
+    // reset messages onChange
+    setXdrError("");
+    setFeeBumpedTx({
+      errors: [],
+      xdr: "",
+    });
+
+    updateBaseFeeInnerXdr(value);
+
+    if (value.length > 0) {
+      const validatedXDR = validate.xdr(value);
+
+      if (validatedXDR?.result && validatedXDR.message) {
+        if (validatedXDR.result === "success") {
+          const result = txHelper.buildFeeBumpTx({
+            innerTxXDR: value,
+            maxFee: fee,
+            sourceAccount: source_account,
+            networkPassphrase: network.passphrase,
+          });
+
+          setFeeBumpedTx(result);
+        } else {
+          setXdrError(validatedXDR.message);
+        }
+      }
+    }
+  };
```

Comment:
We should do similar validation when the page loads if there is XDR. Right now the user ends up in a confusing state because there are no buttons at the bottom.

[lab-fee-bump-refresh.webm](https://github.com/stellar/laboratory/assets/7346473/053f4fd6-e302-49dd-9618-6d7a61a049b4)